Documentation/Signoffs/IdentificationSystem.md

```diff
@@ -0,0 +1,72 @@
+
+The scanner will be used at different ranges to determine an appropriate range for a box to be placed and still be correctly read. To verify the code information, the scanner output will be compared with the expected outcome to ensure a box is not mislabeled. There will be 2 barcode scanners. One will be placed inside the machine so boxes can be loaded. Another will be outside the machine with the RFID reader to scan when a student gets a box. All the modules will be fed through a usb hub.  
+
+To remove data from the computer for use, a program will be added to the MiniPC so when the drive is plugged in, the information is downloaded automatically. The port will be behind a locked compartment so students cannot access the data.  
+
+To connect the Computer to the PLC, a USB NIC will be needed to provide an extra Ethernet port. The PLC uses Profinet, which is a communication protocol used to collect information and control a PLC through Profibus. The Computer will use an application called TIA Portal which allows connection and communication with the PLC and functions as a controller. The program will be created in the TIA portal and loaded to the memory on the PLC. TIA portal also provides diagnostics of the PLC through the software. The PLC the team is using is called a SIMATIC S7 1200 from Siemens and requires certain types of memory card. The three options of memory for the PLC differ only by capacity. The three options for memory size are 2, 4, and 24 MB. Currently the team is considering a 4MB memory card for the PLC. Issues have arrived with a USB NIC not being recognized by TIA Portal. If the software does not recognize the NIC, the way to fix the issue is to enter the IP address manually. 
+
+^ Source: https://us.profinet.com/profinet-explained/  
+
+To connect the PLC to the PC, the TIA Portal software must first be purchased and downloaded. After creating a new project, the device we are using will be added from the available library. The exact CPU is listed in the PLC we received. There is also an option to detect our PLC if we cannot find the device. The portal will use a PN/IE interface and the USB NIC as the adapter for connection. To write values from the sensors, an offset is needed. The common offset of the address is 10. Our team is giving full access to the PLC for the project. Once the PLC is set up, the CPU can be started to run the program. When our program is created, there will need to be a way to give the PLC a command from our code to TIA portal. With the newer versions of TIA portal, python can send data from the program to the portal for the PLC to execute. The data will go through Modbus to reach the TIA portal. Python has libraries such as Pymodbus to connect. This will be used to control the motor and LED indicators. Data can also be read from the PLC if needed. 
```

Comment:
This won't allow you to programmatically interact with the PLC from the PC and vise versa. What you have described is the software needs of the computer used to program the PLC. That is not the same.